buildbuddy.yaml

```diff
@@ -20,9 +20,12 @@ actions:
         branches:
           - "master"
     bazel_commands:
-      # TODO: Fix `test //...` on Mac and do that instead
-      - build //enterprise/server/cmd/executor //enterprise/server //server //cli/cmd/bb --config=mac-workflows
-      - test //enterprise/server/remote_execution/... //cli/... --config=mac-workflows --test_tag_filters=-performance,-webdriver,-docker
+      - |
+        test --config=mac-workflows
+             --
+             //...
+             -//deployment:all
+             -//tools/smarter_device_manager:push_image
```

Comment:
does it work if we mark this `push_image` target with `tags = ["manual"]`?

Reply:
yup, that did the magic